netmon/netmon.go

```diff
@@ -567,26 +567,26 @@ func (n *netmon) handleRTMDelRoute(ev netlink.RouteUpdate) error {
 }
 
 func (n *netmon) handleLinkEvent(ev netlink.LinkUpdate) error {
-	n.logger().Debug("handleLinkEvent: netlink event received")
+	n.logger().Info("handleLinkEvent: netlink event received")
 
 	switch ev.Header.Type {
 	case unix.NLMSG_DONE:
-		n.logger().Debug("NLMSG_DONE")
+		n.logger().Info("NLMSG_DONE")
```

Comment:
How many of these events do we normally get? I'm wondering if `Info` might end up DoS'ing the logs so best to leave at `Debug` level?

Reply:
Yeah, this is only for debug purpose, I ran into some wired problems in CI and I don't have a good environment to reproduce, still struggling...

These changes will be removed in final version. Thanks :-)